resources/js/src/app/components/newsletter/NewsletterInput.vue

```diff
@@ -116,33 +119,38 @@ export default {
 
         save()
         {
-            ApiService.post("/rest/io/customer/newsletter", { email: this.email, firstName: this.firstName, lastName: this.lastName, emailFolder: this.emailFolder, honeypot: this.honeypot })
-                .done(data =>
-                {
-                    if (!!data.containsHoneypot)
+            executeReCaptcha(this.$refs.newsletterForm)
```

Comment:
die Referenz ist hier nicht nötig. Du kannst auch direkt das root Element über `this.$el` in die Funktion geben.